code/src/layouts/best-sellers/BestSellers.tsx

```diff
@@ -0,0 +1,61 @@
+import PageWrapper from "@/layouts/page-wrapper/PageWrapper";
+import AppTypography from "@/components/app-typography/AppTypography";
+import AppBox from "@/components/app-box/AppBox";
+
+import { useGetProductsQuery } from "@/store/api/productsApi";
+
+import { Product } from "@/types/product.types";
+
+import "@/layouts/best-sellers/BestSellers.scss";
+import AppButton from "@/components/app-button/AppButton";
+import AppLink from "@/components/app-link/AppLink";
+
+const BestSellers = () => {
+  const { data: products, isLoading } = useGetProductsQuery();
+
+  if (isLoading) return <AppTypography>Loading...</AppTypography>;
+
+  const renderProducts = (products: Product[]) => {
+    return products.slice(0, 5).map((product) => (
+      <AppLink key={product.id} to="/">
+        <AppBox className="spa-best-sellers__product-container">
+          <img
+            className="spa-best-sellers__img"
+            src={product.image}
+            alt={product.name}
+          />
+          <AppBox>
+            <AppTypography variant="caption">{product.name}</AppTypography>
+            <AppTypography className="spa-best-sellers__price">
+              {product.price}
+            </AppTypography>
+            <AppTypography className="spa-best-sellers__description">
+              {product.description}
+            </AppTypography>
+          </AppBox>
+        </AppBox>
+      </AppLink>
+    ));
```

Comment:
Like I told create separate component for a single product item and pass it as prop

```suggestion
  return products.slice(0, 5).map((product) => (
    <ProductItem key={product.id} product={product} />
  ))
```
Don't forget to create a tests for your implementation here or in another issue as you want